views/js/controller/Delivery/monitoring.js

```diff
@@ -47,6 +48,42 @@ define([
     loadingBar.start();
 
     /**
+     * Format a number to string with leading zeros
+     * @param {Number} n
+     * @param {Number} len
+     * @returns {String}
+     * @private
+     */
+    var _leadingZero = function(n, len) {
+        var value = n.toString();
+        while (value.length < len) {
+            value = '0' + value;
+        }
+        return value;
+    };
+
+    /**
+     * Formats a time value to string
+     * @param {Number} time
+     * @returns {String}
+     * @private
+     */
+    var _timerFormat = function(time) {
+        var seconds = Math.floor(time % 60);
+        var minutes = Math.floor(time / 60) % 60;
+        var hours = Math.floor(time / 3600);
+        var parts = [];
+
+        if (hours) {
+            parts.push(hours);
+        }
+        parts.push(_leadingZero(minutes, 2));
+        parts.push(_leadingZero(seconds, 2));
+
+        return parts.join(':');
```

Comment:
I would suggest truncating the seconds (second level precision is not relevant here since the displayed data is not refresh on real time).
I would also only display minutes, so 1h07 becomes 67min since minutes are more likely to be the most relevant time scale for an item duration.